locale/en/blog/release/v8.0.0.md

```diff
@@ -200,12 +200,12 @@ AssertionError [ERR_ASSERTION]: false == true
 
 Information for static error codes can be queried quickly by referencing the
 Node.js documentation. For instance, the URL to lookup information about the
-`ERR_ASSERTION` error code is
-https://nodejs.org/dist/latest-v7.x/docs/api/errors.html#ERR_ASSERTION.
+`ERR_ARG_NOT_ITERABLE` error code is
```

Comment:
I could have sworn that `ERR_ASSERTION` had been added... will have to make sure it is.

The link format does work tho... try

```https://nodejs.org/dist/latest-v8.x/docs/api/errors.html#ERR_ARG_NOT_ITERABLE
```